diff --git a/bazel/emscripten_toolchain/toolchain.bzl b/bazel/emscripten_toolchain/toolchain.bzl
index 8c724d2..ad49e7e 100644
--- emscripten_toolchain/toolchain.bzl
+++ emscripten_toolchain/toolchain.bzl
@@ -934,7 +934,7 @@ def _impl(ctx):
                 "-iwithsysroot" + "/include/compat",
                 "-iwithsysroot" + "/include",
                 "-isystem",
-                emscripten_dir + "/lib/clang/21/include",
+                emscripten_dir + "/lib/clang/22/include",
             ],
         ),
         # Inputs and outputs
@@ -1101,7 +1101,7 @@ def _impl(ctx):
         emscripten_dir + "/emscripten/cache/sysroot/include/c++/v1",
         emscripten_dir + "/emscripten/cache/sysroot/include/compat",
         emscripten_dir + "/emscripten/cache/sysroot/include",
-        emscripten_dir + "/lib/clang/21/include",
+        emscripten_dir + "/lib/clang/22/include",
     ]
 
     artifact_name_patterns = []
diff --git a/bazel/revisions.bzl b/bazel/revisions.bzl
index a558cce..41da512 100644
--- revisions.bzl
+++ revisions.bzl
@@ -2,6 +2,14 @@
 # DO NOT MODIFY
 
 EMSCRIPTEN_TAGS = {
+    "4.0.13": struct(
+        hash = "32b8ae819674cb42b8ac2191afeb9571e33ad5e2",
+        sha_linux = "2ad887035e3e5cac78abcaeca3b3881897f03b9919d008cbc0ef41d7641247c9",
+        sha_linux_arm64 = "675156bd626c7b19d4f2271ea8ffa77a235d9b0a031116632e53b374cf23754c",
+        sha_mac = "657cbd01e84f0a33cb7a1379baecd18a7d96883222a3a99b43919d8bb2374f55",
+        sha_mac_arm64 = "12ac26e298ef973207eba9332e28da375ec2ba1d32e68e0d8b32de3c886a2e39",
+        sha_win = "a363d6e92dcaf0024d378f1faabb61a139d9152a796f66a475a48e33e90f6adb",
+    ),
     "4.0.3": struct(
         hash = "de2109f0e5e7278d470da11de526aed16c527722",
         sha_linux = "6480f51d0c24130424c696bf83e9774f42246a0109c8d48b59f4520fdfadb928",
